content/en-us/reference/engine/classes/Stats.yaml

```diff
@@ -481,8 +505,10 @@ properties:
     capabilities: []
     writeCapabilities: []
   - name: Stats.UI3DDrawcallCount
-    summary: ''
-    description: ''
+    summary: |
+      A measurement of the number of 3D draw calls made for UI elements in the game's current scene. 
+    description: |
+      The UI3DDrawcallCount property is a measurement of the number of 3D draw calls made for UI elements in the game's current scene. A high count could indicate a high amount of 3D objects being used within UI, potentially hurting performance; however, it is very unlikely you would see a significant count since UI elements are typically 2D.
```

Comment:
```suggestion
      This property is a measurement of the number of 3D draw calls made for UI elements in the game's current scene. A high count could indicate a high amount of 3D objects being used within UI, potentially hurting performance; however, it is very unlikely you would see a significant count since UI elements are typically 2D.
```